src/accelerate/utils/dataclasses.py

```diff
@@ -79,6 +102,27 @@ class AutocastKwargs(KwargsHandler):
     cache_enabled: bool = None
 
 
+class DDPCommunicationHookType(str, BaseEnum):
```

Comment:
Is there a reason this isn't just `BaseEnum`? (Can you check that works)

Reply:
Thank you for the suggestion! I have tested it, and it works correctly using just `BaseEnum`. I will update the code accordingly.